### PR TITLE
Add app-config and app-build to Hooks in README

### PR DESCRIPTION
**Pull request by will-moore**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -84,6 +84,8 @@ overridden by placing an executable file of the same name in an
  * `py-check` runs standard Python quality tools
  * `py-common` runs standard Python package build steps
  * `py-setup` builds and install the Python package
+ * `app-config` configure the app
```
Comment (sbesson) [inline README.md]:
```suggestion
 * `app-config` configures the app
```

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Comment by snoopycrimecop** [comment]
Empty PR description. Please add a short summary of the PR scope and some testing instructions.

**Pull request merged**